e2e/specs/planner.e2e-spec.ts

```diff
@@ -0,0 +1,194 @@
+import { LoginFlow } from '../flows/login.flow';
+import { NavigationFlow } from '../flows/navigation.flow';
+import * as moment from 'moment';
+import { API } from '../utils/api';
+import { SessionCreate } from '../../src/app/sessions/models/session-create.model';
+import { v4 as uuid } from 'uuid';
+import { PlannerPage } from '../pages/planner.po';
+import { DateTimeHelper } from '../utils/date-time-helper';
+import { Judges, JudgesCodes } from '../enums/judges';
+import { RoomCodes, Rooms } from '../enums/rooms';
+import { SessionTypes, SessionTypesCodes } from '../enums/session-types';
+import { Logger } from '../utils/logger';
+import { ElementFinder } from 'protractor';
+import { SessionDetailsDialogPage } from '../pages/session-details-dialog.po';
+
+const loginFlow = new LoginFlow();
+const navigationFlow = new NavigationFlow();
+const plannerPage = new PlannerPage();
+const sessionDetailsDialog = new SessionDetailsDialogPage();
+let today = moment();
+let numberOfVisibleEvents: number;
+let visibleElementsDiff: number;
+let resourceId: string;
+let textToCheck: string;
+let sessionsToCreate: SessionCreate[];
+
+async function clickAndValidateDialog(event: ElementFinder, valuesToCheck: string[]) {
```

Comment:
There's too much implementation details in this spec. It's a good practice to make tests basically a bunch of business steps that are easy to read. Anyone should be able to just take a glance at the test and understand what it does.